bindata/v3.11.0/kube-apiserver/installer-pod.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     app: installer
 spec:
   serviceAccountName: installer-sa
+  priorityClassName: system-node-critical
```

Comment:
This one seems ok. The static pod change looks wrong.

Reply:
The priority to static pod is given to avoid eviction by kubelet in resource crunch scenarios:

https://github.com/kubernetes/kubernetes/blob/a3ccea9d8743f2ff82e41b6c2af6dc2c41dc7b10/pkg/kubelet/eviction/eviction_manager.go#L543-#L549

https://github.com/kubernetes/kubernetes/blob/25cbd1c753fbed15e47c6331ddf043cc00111792/pkg/kubelet/types/pod_update.go#L148

https://github.com/kubernetes/kubernetes/blob/25cbd1c753fbed15e47c6331ddf043cc00111792/pkg/kubelet/types/pod_update.go#L194